scripts/build/travis/configure_travis_trusty.sh

```diff
@@ -1,186 +1,52 @@
 #!/bin/bash
 # You can use your interpreter of choice (bash, sh, zsh, ...)
 
-# For any question please contact:
-#   - kratos@listas.cimne.upc.edu
-#   - croig@cimne.upc.edu
-
-# Or visit our forums at
-#   - http://kratos-wiki.cimne.upc.edu/forum/index.php
-
-########################################################################################################################
-#### CLEANUP                                                                                                        ####
-########################################################################################################################
-# Clean up the terminal
+# For any question please contact with us in:
+#   - https://github.com/KratosMultiphysics/Kratos
+
+# Set compiler
+export CC=gcc-6
+export CXX=g++-6
+
+# Set variables
+export KRATOS_SOURCE="${KRATOS_SOURCE:-${PWD}}"
+export KRATOS_BUILD="${KRATOS_SOURCE}/build"
+export KRATOS_APP_DIR="${KRATOS_SOURCE}/applications"
+
+export KRATOS_BUILD_TYPE=${KRATOS_BUILD_TYPE:="Custom"}
+export KRATOS_APPLICATIONS="${KRATOS_APP_DIR}/ExternalSolversApplication;"
+export KRATOS_APPLICATIONS="${KRATOS_APPLICATIONS}${KRATOS_APP_DIR}/StructuralMechanicsApplication;"
+export KRATOS_APPLICATIONS="${KRATOS_APPLICATIONS}${KRATOS_APP_DIR}/FluidDynamicsApplication;"
+export KRATOS_APPLICATIONS="${KRATOS_APPLICATIONS}${KRATOS_APP_DIR}/MeshMovingApplication;"
+export KRATOS_APPLICATIONS="${KRATOS_APPLICATIONS}${KRATOS_APP_DIR}/DEMApplication;"
+export KRATOS_APPLICATIONS="${KRATOS_APPLICATIONS}${KRATOS_APP_DIR}/CSharpWrapperApplication;"
+export KRATOS_APPLICATIONS="${KRATOS_APPLICATIONS}${KRATOS_APP_DIR}/MetisApplication;"
+export KRATOS_APPLICATIONS="${KRATOS_APPLICATIONS}${KRATOS_APP_DIR}/TrilinosApplication;"
```

Comment:
is this how the configure script will look in the future?
Looks quite complicated compared to the current situation

Reply:
Well you can have a list of applications, like
```export KRATOS_APPLICATIONS="${KRATOS_APP_DIR}/DEMApplication;${KRATOS_APP_DIR}/CSharpWrapperApplication;${KRATOS_APP_DIR}/MetisApplication;..."``` if you prefer.

But I understand you you mean. Another solution is to have a text file with the list of applications that you want to use @pooyan-dadvand what you think about that?

Reply:
I came up with a couple more alternatives. We can also do:

```bash
function add_app {
    export KRATOS_APPLICATIONS="${KRATOS_APPLICATIONS}$1;"
}

add_app ${KRATOS_APP_DIR}/ExternalSolversApplication
add_app ${KRATOS_APP_DIR}/StructuralMechanicsApplication
add_app ${KRATOS_APP_DIR}/FluidDynamicsApplication
# The ${KRATOS_APP_DIR} may be also hidden in the function as an optional parameter.
```

or 

```bash
APPLICATIONS=(
    "${KRATOS_APP_DIR}/ExternalSolversApplication"
    "${KRATOS_APP_DIR}/StructuralMechanicsApplication"
    "${KRATOS_APP_DIR}/FluidDynamicsApplication"
)

export KRATOS_APPLICATIONS=`printf %s";" "${APPLICATIONS[@]}"`
```

Both will work (I have to think the counterparts in win). Now that I code them I think the first one is quite clean. What do you think?

Reply:
Oh ok I was not aware that this PR also changes completely how we compile Kratos :sweat_smile:

Reply:
Yep, we are taking the chance to catch up with what altair already has as well so minimize conflicts

Reply:
could you post an example of how the complete configure file (or whatever it will be) looks like then?

Reply:
```bash
#!/bin/bash
# You can use your interpreter of choice (bash, sh, zsh, ...)

# For any question please contact with us in:
#   - https://github.com/KratosMultiphysics/Kratos

function add_app {
    export KRATOS_APPLICATIONS="${KRATOS_APPLICATIONS}$1;"
}

# Set compiler
export CC=gcc
export CXX=g++

# Set variables
export KRATOS_SOURCE="${KRATOS_SOURCE:-${PWD}}"
export KRATOS_BUILD="${KRATOS_SOURCE}/build"
export KRATOS_APP_DIR="${KRATOS_SOURCE}/applications"

# Set applications to compile
export KRATOS_BUILD_TYPE=${KRATOS_BUILD_TYPE:="Release"}

APPLICATIONS=(
    "${KRATOS_APP_DIR}/ExternalSolversApplication"
    "${KRATOS_APP_DIR}/StructuralMechanicsApplication"
    "${KRATOS_APP_DIR}/FluidDynamicsApplication"
)

export KRATOS_APPLICATIONS=`printf %s";" "${APPLICATIONS[@]}"`
echo $KRATOS_APPLICATIONS

function add_app {
    export KRATOS_APPLICATIONS="${KRATOS_APPLICATIONS}$1;"
}

add_app ${KRATOS_APP_DIR}/ExternalSolversApplication
add_app ${KRATOS_APP_DIR}/StructuralMechanicsApplication
add_app ${KRATOS_APP_DIR}/FluidDynamicsApplication

# Clean
clear
rm -rf "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}/cmake_install.cmake"
rm -rf "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}/CMakeCache.txt"
rm -rf "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}/CMakeFiles"

# Configure
cmake -H"${KRATOS_SOURCE}" -B"${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}" \
-DINSTALL_PYTHON_USING_LINKS=ON \
-DUSE_COTIRE=ON

# Buid
cmake --build "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}" --target all_unity -- -j4
cmake --build "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}" --target install/fast -- -j4 
```